public/app/core/components/Branding/Branding.tsx

```diff
@@ -41,13 +53,6 @@ export class Branding {
   static AppTitle = 'Grafana';
   static LoginTitle = 'Welcome to Grafana';
   static GetLoginSubTitle = () => {
```

Comment:
This needs an explicit return now `static GetLoginSubTitle = (): string | null => {`, otherwise TS won't allow returning anything else than `null` here (this breaks in enterprise).